File: data/imported/knowledge/slides/java/merged/snippets/snippet_008.java
@Test
    void testInsertComment() {
        try (Presentation pres = new Presentation()) {
            ICommentAuthor author = pres.getCommentAuthors().addAuthor("Alice", "A");
            ISlide slide = pres.getSlides().get(0);
            LocalDateTime now = LocalDateTime.now();
            author.getComments().addComment("First", slide, new PointF(1, 1), now);
            author.getComments().addComment("Third", slide, new PointF(1, 3), now);
            author.getComments().insertComment(1, "Second", slide, new PointF(1, 2), now);
            assertThat(author.getComments().size()).isEqualTo(3);
            assertThat(author.getComments().get(1).getText()).isEqualTo("Second");
        }
    }

File: data/imported/knowledge/slides/net/merged/snippets/snippet_008.cs
[Fact]
    public void TestInsertComment()
    {
        // insert_comment places at the correct index.
        using var pres = new Presentation();
        var author = pres.CommentAuthors.AddAuthor("Alice", "A");
        var slide = pres.Slides[0];
        var now = DateTime.Now;
        author.Comments.AddComment("First", slide, new PointF(1, 1), now);
        author.Comments.AddComment("Third", slide, new PointF(1, 3), now);
        author.Comments.InsertComment(1, "Second", slide, new PointF(1, 2), now);
        author.Comments.Count.Should().Be(3);
        author.Comments[1].Text.Should().Be("Second");
    }